### PR TITLE
init cdk lambda layer example

### PR DESCRIPTION
**Pull request by kien-cs**

### CODE REVIEW

File: cdk-lambda-layer-example/lib/cdk-lambda-layer-example-stack.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      functionName: 'CodeSee-Lambda-Layer-Example',
+      code: lambda.Code.fromAsset('./src'),
+      handler: 'index.handler',
+      runtime: lambda.Runtime.NODEJS_16_X,
+      layers: [
+        lambda.LayerVersion.fromLayerVersionArn(this, 'CodeSeeLayer', 'arn:aws:lambda:us-east-2:666523192759:layer:codesee-dd-bridge:8'),
+      ]
+    });
+
+    datadog.addLambdaFunctions([exampleLambda])
```
Comment (jleven) [inline cdk-lambda-layer-example/lib/cdk-lambda-layer-example-stack.ts:29]: This looks a lot like what we tried earlier today, but there was no Datadog Layer on the deployed lambda. Did you verify both layers are present? Do you have any idea why we were seeing only the CodeSee layer before?

Comment (kien-cs) [reply cdk-lambda-layer-example/lib/cdk-lambda-layer-example-stack.ts:29]: tagged you in the thread https://codesee.slack.com/archives/C04SEN84VCG/p1677629816001899?thread_ts=1677622945.512659&cid=C04SEN84VCG

**Review by jleven — APPROVED** [review]
Thanks, looks great!

File: cdk-lambda-layer-example/lib/cdk-lambda-layer-example-stack.ts (hunk 1 above)
Comment (kien-cs) [reply cdk-lambda-layer-example/lib/cdk-lambda-layer-example-stack.ts:29]: Verified both layers are present. No idea why we are not seeing, probably sst framework conflict, I'll check that next.

**Pull request merged**